Tests/DbgEngTest/TestBase.cs

```diff
@@ -23,18 +23,17 @@ internal static void SyncStop()
             System.Threading.Monitor.Exit(synchronizationObject);
         }
 
-
         /// <summary>
-        /// Initializes the test class with the specified dump file.
+        /// Creates absolute paths out of givn file path and symbol path.
         /// </summary>
-        /// <param name="dumpFile">The dump file.</param>
-        /// <param name="symbolPath">The symbol path.</param>
-        /// <param name="addSymbolServer">if set to <c>true</c> symbol server will be added to the symbol path.</param>
-        protected static void Initialize(string dumpFile, string symbolPath, bool addSymbolServer = true)
+        /// <param name="dumpFile"></param>
+        /// <param name="symbolPath"></param>
+        /// <param name="addSymbolServer"></param>
```

Comment:
dumpFile got renamed. Please update comment...

---
In reply to: [69812242](https://github.com/southpolenator/WinDbgCs/pull/8#discussion_r69812242) [](ancestors = 69812242,69627632)